ios/push-notifications.md

```diff
@@ -67,50 +60,107 @@ Use the following steps to connect add push notification backend services to you
 
     Run `pod install --repo-update` before you continue.
 
-    If you encounter an error message that begins `[!] Failed to connect to GitHub to update the CocoaPods/Specs...`, and your internet connectivity is working, you may need to [update openssl and Ruby](https://stackoverflow.com/questions/38993527/cocoapods-failed-to-connect-to-github-to-update-the-cocoapods-specs-specs-repo/48962041#48962041).
-
-1. Classes that call Amazon Pinpoint APIs must use the following import statements:
+1. Add the following import statements:
```

Comment:
Add the following import statements to your AppDelegate file

Reply:
updating